CPS.md

```diff
@@ -540,7 +538,7 @@ Certificates may be administratively revoked by ISRG if it is determined that th
 
 Revocation requests may be made by anyone, at any time, via the Certificate Revocation interface of the ACME Protocol defined in RFC 8555 section 7.6. Successful revocation requests with a reason code of `keyCompromise` will result in the affected key being blocked for future issuance and all currently valid certificates with that key will be revoked, regardless of whether compromise was demonstrated per the requirements in Section 4.9.12 of this document.
 
-Requests for revocation may also be made by emailing [cert-prob-reports@letsencrypt.org](mailto:cert-prob-reports@letsencrypt.org). ISRG will respond to such requests within 24 hours, though an investigation into the legitimacy of the request may take longer.
+Requests for revocation may also be made by emailing [cert-prob-reports@letsencrypt.org](mailto:cert-prob-reports@letsencrypt.org). ISRG maintains a continuous (24x7) ability to accept and respond to revocation requests and related inquiries. ISRG will respond to such requests within 24 hours, though an investigation into the legitimacy of the request may take longer.
```

Comment:
This just copies the sentence from the other section verbatim. As the issue reporter suggested, I think we should change this language to match the CP/BRs exactly: "...a continuous 24x7 ability to accept and respond to revocation requests and Certificate Problem Reports."

Reply:
Good call, done.